### PR TITLE
More improvements.

### PR DESCRIPTION
**Pull request by terabytesoftw**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ✔️

- [x] Add documentation.
- [x] Resolve issues static analisys `PSALM` depends https://github.com/yiisoft/db/pull/171.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,16 +6,15 @@
...
-[![Total Downloads](https://poser.pugx.org/yiisoft/yii-db-migration/downloads.png)](https://packagist.org/packages/yiisoft/yii-db-migration)
-[![Build status](https://github.com/yiisoft/yii-db-migration/workflows/build/badge.svg)](https://github.com/yiisoft/yii-db-migration/actions?query=workflow%3Abuild)
-[![Scrutinizer Code Quality](https://scrutinizer-ci.com/g/yiisoft/yii-db-migration/badges/quality-score.png?b=master)](https://scrutinizer-ci.com/g/yiisoft/yii-db-migration/?branch=master)
-[![Code Coverage](https://scrutinizer-ci.com/g/yiisoft/yii-db-migration/badges/coverage.png?b=master)](https://scrutinizer-ci.com/g/yiisoft/yii-db-migration/?branch=master)
-[![Mutation testing badge](https://img.shields.io/endpoint?style=flat&url=https%3A%2F%2Fbadge-api.stryker-mutator.io%2Fgithub.com%2Fyiisoft%2Fyii-db-migration%2Fmaster)](https://dashboard.stryker-mutator.io/reports/github.com/yiisoft/yii-db-migration/master)
-[![static analysis](https://github.com/yiisoft/yii-db-migration/workflows/static%20analysis/badge.svg)](https://github.com/yiisoft/yii-db-migration/actions?query=workflow%3A%22static+analysis%22)
+| Package     |
```
Comment (samdark) [inline README.md:11]: I liked it more when there were just badges without division on package / actions.

**Review by samdark — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -25,26 +24,179 @@ The package could be installed via composer:
...
+
+```
+Available commands:
+  database/list    Lists all tables in the database.
+  generate/create  Generate migration template.
+  help             Displays help for a command
+  list             Lists commands
+  migrate/down     Downgrades the application by reverting old migrations.
+  migrate/history  Displays the migration history.
+  migrate/new      Displays the migration history.
```
Comment (roxblnfk) [inline README.md:145]: Description is a duplicate of `migrate/history`

File: src/Command/HistoryCommand.php [hunk 3]
```diff
@@ -50,10 +50,14 @@ public function configure(): void
             ->setHelp('This command displays the migration history.');
     }
 
+    /**
+     * @return int
+     */
```
Comment (roxblnfk) [inline src/Command/HistoryCommand.php:55]:
```suggestion
```

File: src/Command/NewCommand.php [hunk 4]
```diff
@@ -50,14 +50,16 @@ public function configure(): void
             ->setHelp('This command displays the migration history.');
     }
 
+    /**
+     * @return int
+     */
```
Comment (roxblnfk) [inline src/Command/NewCommand.php:55]:
```suggestion
```

File: src/Command/RedoCommand.php [hunk 5]
```diff
@@ -62,10 +62,14 @@ public function configure(): void
             ->setHelp('This command redoes the last few migrations.');
     }
 
+    /**
+     * @return int
+     */
```
Comment (roxblnfk) [inline src/Command/RedoCommand.php:67]:
```suggestion
```

File: src/Command/UpdateCommand.php [hunk 6]
```diff
@@ -56,13 +56,17 @@ public function configure(): void
             ->setHelp('This command applying new migrations to database.');
     }
 
+    /**
+     * @return int
+     */
```
Comment (roxblnfk) [inline src/Command/UpdateCommand.php:61]:
```suggestion
```

File: README.md (hunk 2 above)
Comment (terabytesoftw) [reply README.md:145]: fixed.

File: src/Command/UpdateCommand.php (hunk 6 above)
Comment (terabytesoftw) [reply src/Command/UpdateCommand.php:61]: fixed.

File: src/Command/RedoCommand.php (hunk 5 above)
Comment (terabytesoftw) [reply src/Command/RedoCommand.php:67]: Fixed.

File: src/Command/NewCommand.php (hunk 4 above)
Comment (terabytesoftw) [reply src/Command/NewCommand.php:55]: fixed.

File: src/Command/HistoryCommand.php (hunk 3 above)
Comment (terabytesoftw) [reply src/Command/HistoryCommand.php:55]: fixed.

File: README.md (hunk 1 above)
Comment (terabytesoftw) [reply README.md:11]: done revert readme.md badges.

**Review by samdark — APPROVED** [review]

**Pull request merged**